gramps/gui/views/listview.py

```diff
@@ -1071,6 +1071,10 @@ def export(self, obj):
         combobox.set_active(0)
         box.show_all()
         chooser.set_extra_widget(box)
+        default_dir = config.get('paths.report-directory')
+        if not default_dir:
```

Comment:
Is this necessary?  The ['paths.report-directory'](https://github.com/gramps-project/gramps/blob/master/gramps/gen/config.py#L228) defaults to `USER_HOME` anyway.